examples/xlnet/run.py

```diff
@@ -0,0 +1,249 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
```

Comment:
Rename this file to sth like `xlnet_classification_main.py` as with other examples